style.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,300;0,400;1,300&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;1,100;1,300;1,400;1,500;1,700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+
+  /* overflow: hidden; */
+}
+
+.toolbar_container {
+  display: flex;
+  flex-direction: row;
+  background-color: #3c3a39;
+  max-width: 100%;
+  align-items: center;
+  justify-content: center;
+  align-content: center;
+  justify-items: center;
+  gap: 180px;
+  height: 17vh;
+}
```

Comment:
- [ ] Kindly consider doing the following to make your design identical to the figma design.
   -  We try to avoid giving exact widths and heights to containers and elements because they makes responsive design almost impossible to achieve. instead we try to apply margin and paddings. I would recommend you to remove the `height` and instead apply padding top and bottom to it.
   - Try changing `justify-content: center;` to `justify-content: space-between;`. 
   - Also I dont think there is any need for the `gap` property here.